keystorage/src/main/java/tech/pegasys/web3signer/keystorage/azure/AzureKeyVault.java

```diff
@@ -103,6 +115,43 @@ public CryptographyClient fetchKey(final String keyName, final String keyVersion
         .buildClient();
   }
 
+  public HttpRequest getRemoteSigningHttpRequest(
```

Comment:
Why are we returning a HttpRequest rather having a method to just sign the request. The Azure logic is split between the `AzureKeyVault` and the `AzureVaultSigner`. I see that there was specific Azure code already in the signer but just trying understand the split/scope for the `AzureKeyVault`

Reply:
This is some tech debt that we could probably clean up once we tide up the AzureKeyVault class but for now I can't find a way to do it without exposing the tokenCredential which I think is a worse solution than just asking the AzureKeyVault to assemble the signing http request.